### ISSUE TITLE
Very slow transfers using PS4 Site connection to share (NFS or SMB)

### ISSUE THREAD

**Issue by HVR88** [cy33hc/ps4-ezremote-client#20]
I'm seeing very slow transfer speeds while installing PKG from NFS or SMB share using only the UI on the PS4. I can't measure exactly how slow without a transfer speed display, but it's at least 5-10x (or more) slower than using the web UI to enter a direct HTTP URL to the same file (running HTTP file server on my UNRAID network server).

It's also much slower than using a PC/Mac app to directly send or proxy the same file with a double hop using GoldHen's binloader or RPI (NAS share -> Mac -> PS4) - the app is Direct Package Installer (here: https://github.com/marcussacana/DirectPackageInstaller)

**Comment by cy33hc** [comment]
@HVR88 I acknowledge it is slower via nfs/smb and I have some ideas on improving speed, hope to implement that pretty soon. Currently concentrating on porting some of the functions in the PS4 app to the ps3/vita and switch.

**Comment by cy33hc** [comment]
@HVR88 I have made huge improvements to the install speed for SMB, NFS. Its still not as fast as via HTTP and WebDav. Check new release https://github.com/cy33hc/ps4-ezremote-client/releases/tag/1.24

**Comment by HVR88** [comment]
I'm finally able to get back to some testing and I found what looks like a discrepancy in speed depending on HOW a link is added rather than the transport it's using. It's significant too, possibly 10x or more difference in speed.

I set up Apache httpd on my local server and configured the web directory to enable indexes and so I can serve local files easily.

From the PS4:
• Listing of files displayed when browsing to the web directory from EZ on the PS4: http://192.168.1.3
• Select a 4GB pkg file and install - speed is relatively slow

By comparison, from my computer, while accessing the PS4's EZ remote UI:
• Using the Install URL command, I paste the link http://192.168.1.3/filename.pkg
• installation starts on PS4 and the speed is at least an order of magnitude faster than the previous test
• a 4GB file installs in seconds
